test/api/v01/with_solver_test.rb

```diff
@@ -74,4 +74,33 @@ def test_returned_graph
     end
     delete_completed_job @job_id, api_key: 'ortools' if @job_id
   end
+
+  def test_filter_intermediate_solution
+    # Verify that intermediate solution are correctly filtered
+    asynchronously start_worker: true do
+      vrp = VRP.lat_lon
+      vrp[:configuration][:resolution][:duration] = 2000
+      vrp[:units] = [{id: 'unit_1'}]
+      vrp[:services].each{ |s|
+        s[:quantities] = [{unit_id: 'unit_1', value: 1}]
```

Comment:
The project convention should follow the "space" rule of the SpaceInsideHashLiteralBraces cop https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/SpaceInsideHashLiteralBraces

It probably has been deactivated through the mapotempo_rubocop gem :/

```suggestion
        s[:quantities] = [{ unit_id: 'unit_1', value: 1 }]
```